### PR TITLE
Feature/in 910 virus scan

### PR DESCRIPTION
**Pull request by lorenzoAtBEIS**
Replacing ClamAV virus scanner with Cloudmersive virus scanner api. 
**This PR contains a free developer API key for dev purposes only**

### CODE REVIEW

**Review by BEISKRAM — APPROVED** [review]

**Review by BEISKRAM — APPROVED** [review]

**Review by BEISKRAM — APPROVED** [review]

File: INZFS.MVC/Services/FileUpload/FileUploadService.cs [hunk 1]
```diff
@@ -87,13 +84,20 @@ public async Task<string> Validate(IFormFile file)
             }
             
             //TODO : Switch to virus scanning service
-            var containsVirus = false; // await _virusScanService.ScanFile(file);
-            if (containsVirus)
+            var containsVirus = _virusScanService.ScanFile(file);
+            if (containsVirus == "Virus detected")
```
Comment (sunilbeis) [inline INZFS.MVC/Services/FileUpload/FileUploadService.cs:88]: Can't this return a boolean value instead of text ?

Comment (sunilbeis) [reply INZFS.MVC/Services/FileUpload/FileUploadService.cs:88]: Or is that if a string is returned, then the file has some issue, in which case the string can be simply returned as it.

File: INZFS.MVC/Services/VirusScan/VirusScanService.cs [hunk 2]
```diff
@@ -1,86 +1,58 @@
...
+        
+            var apiInstance = new ScanApi();
+            var inputFile = file.OpenReadStream(); 
+            var allowExecutables = false;  
+            var allowInvalidFiles = false;  
+            var allowScripts = false;  
+            var allowPasswordProtectedFiles = false;  
+            var allowMacros = true;  
+            var allowXmlExternalEntities = true;  
+            var restrictFileTypes = ".xlsx,.xlx,.pdf,.doc,.docx";
```
Comment (sunilbeis) [inline INZFS.MVC/Services/VirusScan/VirusScanService.cs:31]: I think, this list could change depending uopn the file. Does it matter ?

**Review by BEISKRAM — APPROVED** [review]

**Review by BEISKRAM — DISMISSED** [review]

**Review by sunilbeis — APPROVED** [review]

**Review by sunilbeis — APPROVED** [review]

**Comment by lorenzoAtBEIS** [comment]
Waiting for DSU approval - closing for now

**Pull request closed without merging**